hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/hudi/HoodieBootstrapRelation.scala

```diff
@@ -115,8 +113,8 @@ abstract class BaseHoodieBootstrapRelation(override val sqlContext: SQLContext,
     fileSlices.map { fileSlice =>
       val baseFile = fileSlice.getBaseFile.get()
       if (baseFile.getBootstrapBaseFile.isPresent) {
-        val partitionValues = getPartitionColumnsAsInternalRowInternal(baseFile.getBootstrapBaseFile.get.getPathInfo,
-          bootstrapBasePath, extractPartitionValuesFromPartitionPath = isPartitioned)
+        val partitionValues = getPartitionColumnsAsInternalRowInternal(baseFile.getPathInfo,
```

Comment:
Should this use `baseFile.getBootstrapBaseFile`?

Reply:
No, we are just going to use the regular base path because that is what we will be doing if we ever do any updates, or compaction, or clustering anyway. So I think this is a better solution than trying to parse the source table path correctly.